cmd/kubeless/function_test.go

```diff
@@ -217,4 +218,65 @@ func TestGetFunctionDescription(t *testing.T) {
 		t.Error("Unexpected result")
 	}
 
+	// It should upload the function if Minio is available
+	minioFakeSvc := v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "kubeless",
+			Name:      "minio",
+		},
+	}
+	cli := fake.NewSimpleClientset(&minioFakeSvc)
+	result4, err := getFunctionDescription("test", "default", "file.handler", file.Name(), "dependencies", "runtime", "", "", "test-image", "128Mi", true, []string{"TEST=1"}, []string{"test=1"}, spec.Function{}, cli, rest.RESTClient{})
```

Comment:
I am stuck with this, I am not able to mock or obtain a fake RESTClient, I tried with `"k8s.io/client-go/rest/fake"` but it causes a `panic` when trying to Post. Do you have any experience with that @anguslees ?